src/App.tsx

```diff
@@ -15,48 +16,128 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE = 'none',
+  ALPHABET = 'alphabet',
+  LENGTH = 'length',
+}
+
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  const visibleGoods = [...goods];
+
+  visibleGoods.sort((a, b) => {
+    switch (sortType) {
+      case 'none':
+        return 0;
+      case 'alphabet':
+        return a.localeCompare(b);
+      case 'length':
+        return a.length - b.length;
+      default:
+        return 0;
+    }
```

Comment:
default and none cases can be merged. also let's use the power of typescript and check types better
```suggestion
    switch (sortType) {
        return 0;
      case SortTyp.ALPHABET:
        return a.localeCompare(b);
      case SortTyp.LENGTH:
        return a.length - b.length;
      case SortType.NONE:
      default:
        return 0;
    }
```